system/Common.php

```diff
@@ -598,7 +598,7 @@ function helper($filenames): void
             if (str_contains($filename, '\\')) {
                 $path = $loader->locateFile($filename, 'Helpers');
 
-                if ($path !== '') {
+                if (!$path) {
```

Comment:
should be strict empty string or false

```suggestion
                if ($path === false || $path === '') {
```

Reply:
Ok, `$path === false` should be enough, since empty string should never happen


```suggestion
                if ($path === false) {
```